connect/runtime/src/test/java/org/apache/kafka/connect/integration/RebalanceSourceConnectorsIntegrationTest.java

```diff
@@ -270,7 +270,6 @@ public void testRemovingWorker() throws Exception {
                 WORKER_SETUP_DURATION_MS, "Connect and tasks are imbalanced between the workers.");
     }
 
-    @Ignore // TODO: To be re-enabled once we can make it less flaky (KAFKA-12495, KAFKA-12283)
```

Comment:
@vamossagar12 , in L191, there's also a disabled test, I think after this patch, they should be reliable.

Reply:
done.